application/src/main/java/org/opentripplanner/api/parameter/ApiRequestMode.java

```diff
@@ -12,7 +13,8 @@ public enum ApiRequestMode {
   TRAM(TransitMode.TRAM),
   SUBWAY(TransitMode.SUBWAY),
   RAIL(TransitMode.RAIL),
-  BUS(TransitMode.BUS, TransitMode.COACH),
+  BUS(TransitMode.BUS),
+  COACH(TransitMode.COACH),
```

Comment:
There has been some discussion if this have an effect on the TransModel API, but it does not. The ApiRequestMode and the Qualified mode classes is not in use in the Transmodel API.